File: src/main/java/no/nav/veilarboppfolging/controller/response/Veilarbportefoljeinfo.java
package no.nav.veilarboppfolging.controller.response;



import no.nav.common.types.identer.AktorId;
import no.nav.common.types.identer.NavIdent;

import java.time.ZonedDateTime;


public class Veilarbportefoljeinfo {
    private AktorId aktorId;

    private NavIdent veilederId;
    private boolean erUnderOppfolging;
    private boolean nyForVeileder;
    private boolean erManuell;
    private ZonedDateTime startDato;
    private ZonedDateTime tilordnetTidspunkt;
}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/response/Veilarbportefoljeinfo.kt
package no.nav.veilarboppfolging.controller.response

import no.nav.common.types.identer.AktorId
import no.nav.common.types.identer.NavIdent
import java.time.ZonedDateTime


data class Veilarbportefoljeinfo(
    val aktorId: AktorId,
    val veilederId: NavIdent?,
    val erUnderOppfolging: Boolean,
    val nyForVeileder: Boolean,
    val erManuell: Boolean,
    val startDato: ZonedDateTime? = null,
    val tilordnetTidspunkt: ZonedDateTime? = null
)
